app/assets/stylesheets/fontawesome_icons.sass

```diff
@@ -1,5 +1,4 @@
 [class^="icon-"], [class*=" icon-"]
-  font-family: FontAwesome
```

Comment:
are we sure about this? wouldn't it affect across all the pages if it's removed?

Reply:
it would, good point.   We are trying to use Roboto font across all pages, but, agreed that this may introduce unintended behavior elsewhere.  will refactor so that the styling for the commits pages is separate from the global styling.